packages/web-components/src/components/masthead/__stories__/masthead.stories.ts

```diff
@@ -96,7 +97,7 @@ export default {
     props: {
       'dds-masthead-container': {
         // Lets `<dds-masthead-container>` load the nav links if `CORS_PROXY` is set
-        navLinks: process.env.CORS_PROXY ? undefined : links,
+        navLinks: process.env.CORS_PROXY && !inPercy() ? undefined : links,
```

Comment:
Does this mean we don't want to use mock version of masthead links in Percy...?

Reply:
we actually do want to use the mocked version in percy. I think with this it means that we are indeed using the mocked version, `CORS_PROXY` is defined, and `inPercy()` returns true.

Reply:
Got it, my bad... Thanks @jeffchew for clarifying!